apispec/api.spec.json

```diff
@@ -0,0 +1,99 @@
+{
+    "openapi": "3.0.0",
+    "servers": [
+        {
+            "description": "Local entitlements API",
+            "url": "/api/entitlements/v1/"
+        },
+        {
+            "description": "CI entitlements API",
+            "url": "https://ci.cloud.paas.upshift.redhat.com/api/entitlements/v1/"
+        },
+        {
+            "description": "QA entitlements API",
+            "url": "https://qa.cloud.paas.upshift.redhat.com/api/entitlements/v1/"
+        },
+        {
+            "description": "PROD entitlements API",
+            "url": "https://cloud.redhat.com/api/entitlements/v1/"
+        }
+    ],
+    "info": {
+        "description": "Service for determining subscriptions in cloud management\n[cloud.redhat.com](http://cloud.redhat.com/api/v1/entitlements)\n",
+        "version": "1.0.0",
+        "title": "Entitlements",
+        "contact": {
+            "email": "ihands@redhat.com"
+        },
+        "license": {
+            "name": "Apache 2.0",
+            "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+        }
+    },
+    "tags": [
+        {
+            "name": "entitlements",
+            "description": "querying your entitlements"
+        },
+        {
+            "name": "api",
+            "description": "RESTful service"
+        },
+        {
+            "name": "permissions",
+            "description": "Operations about user"
```

Comment:
What does `Operations about user` mean?

Reply:
@kruai I am not completely sure, tbh. I stole the skeleton from the old TypeScript api and made minimal changes to reflect the go api :grimacing:

Reply:
OK, that's fine. If it's fields we already set, I don't really see that as a change.